django/db/backends/base/schema.py

```diff
@@ -200,6 +200,13 @@ def effective_default(self, field):
                 default = six.binary_type()
             else:
                 default = six.text_type()
+        elif getattr(field, 'auto_now', False) or getattr(field, 'auto_now_add', False):
+            default = timezone.now()  # default for DateTimeField
+            internal_type = field.get_internal_type()
+            if internal_type == 'DateField':
+                default = default.date()
```

Comment:
FWIW, I wasn't sure we should even fix this ticket since auto_now/auto_now_add are somewhat weird APIs (see #22995). I'm a bit wary of changing them to be tz aware since there would be some inconsistency in existing user's timestamps on upgrade.